pyproject.toml

```diff
@@ -1,2 +1,34 @@
 [tool.black]
-target_version = ['py38']
\ No newline at end of file
+target_version = ['py38']
+
+[tool.poetry]
+name = "eq-questionnaire-validator"
+version = "0.1.0"
```

Comment:
Maybe worth removing since this isn't a packahe.

Reply:
It is required
 
 ```
RuntimeError

  The Poetry configuration is invalid:
    - 'name' is a required property
    - 'version' is a required property
```